docs/cisco.dcnm.dcnm_fabric_module.rst

```diff
@@ -51,6 +51,2965 @@ Parameters
             </tr>
                                 <tr>
                     <td class="elbow-placeholder"></td>
+                <td colspan="2">
```

Comment:
Looks like the documentation prep collection was not run after a bunch of new properties were added to the fabric module.  Good that they are being added here.